dependencies/CMakeLists.txt

```diff
@@ -1,74 +1,79 @@
 # Initializes a git submodule if it hasn't been initialized before
-# Does NOT attempt to update or otherwise modify git submodules that are already initialized.
-function(initialize_submodule DIRECTORY)
-  if(NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${DIRECTORY}/.git)
-    find_package(Git QUIET REQUIRED)
-    message(STATUS "${CMAKE_CURRENT_SOURCE_DIR}/${DIRECTORY}/.git does not exist. Initializing ${DIRECTORY} submodule ...")
-    execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init ${CMAKE_CURRENT_SOURCE_DIR}/${DIRECTORY}
-                    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-                    RESULT_VARIABLE GIT_EXIT_CODE)
-    if(NOT GIT_EXIT_CODE EQUAL "0")
-      message(FATAL_ERROR "${GIT_EXECUTABLE} submodule update --init dependencies/${DIRECTORY} failed with exit code ${GIT_EXIT_CODE}, please checkout submodules")
+
+
+find_package(Git QUIET) # We want the library to build even if git is missing
+if (Git_FOUND)
```

Comment:
We should probably tell people if git is not found and they specified SIMDJSON_GOOGLE_BENCHMARKS or SIMDJSON_COMPETITION. More of a warning though, and they can disable the warning by setting these OFF.

Reply:
I have added a bunch of message STATUS thingies that make this transparent.